test/provider/AppDirectory.unittest.ts

```diff
@@ -88,16 +96,35 @@ beforeEach(() => {
     });
 });
 
+beforeAll(async () => {
+    const store = new Store<ConfigurationObject>(require('../../gen/provider/config/defaults.json'));
+    Injector.rebind<'ENVIRONMENT'>(Inject.ENVIRONMENT).toConstantValue(createMockEnvironmnent() as Environment);
+    Injector.rebind<'CONFIG_STORE'>(Inject.CONFIG_STORE).toConstantValue({
+        config: store
+    });
+    Injector.rebind<'RESOLVER'>(Inject.RESOLVER).toConstantValue({
+        handleIntent: async (intent: Intent): Promise<ResolverResult> => {
+            return {app: null!};
+        },
+        cancel: async (): Promise<void> => {}
+    });
+
+    await Injector.init();
+});
+
 describe('AppDirectory Unit Tests', () => {
     describe('When querying the Directory', () => {
         beforeEach(() => {
             mockJson.mockResolvedValue(fakeApps);
+            const configStore = Injector.get<'CONFIG_STORE'>(Inject.CONFIG_STORE);
+            configStore.config.add({level: 'desktop'}, {applicationDirectory: DEV_APP_DIRECTORY_URL});
```

Comment:
Suggest set this in the beforeAll, before AppDirectory is created, and then also have some tests specifically around this changing

Reply:
We can add this in as part of SERVICE-516 as we discussed